src/algebra/lie/basic.lean

```diff
@@ -572,39 +574,53 @@ instance lie_subalgebra_lie_algebra (L' : lie_subalgebra R L) : lie_algebra R L'
 
 namespace lie_subalgebra
 
-@[simp] lemma mem_coe {L' : lie_subalgebra R L} {x : L} :
-  x ∈ (L' : set L) ↔ x ∈ L' := iff.rfl
+variables {R L} {L' : lie_subalgebra R L}
+
+@[simp] lemma zero_mem : (0 : L) ∈ L' := (L' : submodule R L).zero_mem
+
+lemma smul_mem {t : R} {x : L} (h : x ∈ L') : t • x ∈ L' := (L' : submodule R L).smul_mem t h
```

Comment:
Given `t` is an explicit argument to `submodule` it probably should be here too:
```suggestion
lemma smul_mem (t : R){x : L} (h : x ∈ L') : t • x ∈ L' := (L' : submodule R L).smul_mem t h
```

Reply:
Yep, definitely. Thanks.